pkg/parser/parser_epub.go

```diff
@@ -8,10 +8,21 @@ import (
 	"github.com/readium/go-toolkit/pkg/mediatype"
 	"github.com/readium/go-toolkit/pkg/parser/epub"
 	"github.com/readium/go-toolkit/pkg/pub"
+	"github.com/readium/go-toolkit/pkg/service"
 	"github.com/readium/go-toolkit/pkg/util"
 )
 
 type EPUBParser struct {
```

Comment:
Since we now have an `epub` package, maybe this could be moved under `epub.Parser`?

Reply:
Done